### PR TITLE
Feat script file testing a user route configuration

### PR DESCRIPTION
**Pull request by ArvsIndrarys**

### CODE REVIEW

**Comment by mirza99** [comment]
How to use it ? It'll be nice to add a "how to test your config" in the readme.

**Comment by ArvsIndrarys** [comment]
Done @mirza99

File: README.md [hunk 1]
```diff
@@ -87,9 +87,18 @@ console.log(routing.generate('persons.findAll'));
 console.log(routing.generate('persons.findById', {'id': 1}));
 ```
 
+#### Testing your config file
+
+You can test your `routes.js` file by using `tester.js` in the lib folder.
+For example :
+```
+cd lib
```
Comment (mirza99) [inline README.md]: It's `cd node_modules/@meligys/routing/lib`

**Review by mirza99 — CHANGES_REQUESTED** [review]

**Comment by ArvsIndrarys** [comment]
Changes done @mirza99

**Review by mirza99 — APPROVED** [review]

**Pull request merged**